docs/index.md

```diff
@@ -54,121 +54,6 @@ import * as fcl from "@onflow/fcl";
 ```js
 const fcl = require("@onflow/fcl");
 ```
----
-## FCL for Dapps
-#### Wallet Interactions
-
-- *Wallet Discovery* and *Sign-up/Login*: Onboard users with ease. Never worry about supporting multiple wallets. 
-Authenticate users with any [FCL compatible wallet](#current-wallet-providers).
-```js
-// in the browser
-import * as fcl from "@onflow/fcl"
-
-fcl.config({
-  "discovery.wallet": "https://fcl-discovery.onflow.org/testnet/authn", // Endpoint set to Testnet
-})
-
-fcl.authenticate()
-```
-![FCL Default Discovery UI](images/fcl-discovery-screenshot.jpg)
-
-
-- *Interact with smart contracts*: Authorize transactions via the user's chosen wallet 
-- *Prove ownership of a wallet address*: Signing and verifying user signed data
-
-[Learn more about wallet interactions >](https://docs.onflow.org/fcl/reference/api/#wallet-interactions)
-
-#### Blockchain Interactions
```

Comment:
This is not specific to Discovery. Why is this being removed?

Reply:
oh, that is a good point. @wise4rmgod some of the content here has to stay

Reply:
Alright